presto-main/src/main/java/com/facebook/presto/operator/TwoChannelJoinProbe.java

```diff
@@ -27,11 +27,7 @@
 package com.facebook.presto.operator;
 
 import com.facebook.presto.spi.Page;
-import com.facebook.presto.spi.PageBuilder;
 import com.facebook.presto.spi.block.Block;
-import com.facebook.presto.spi.type.Type;
-
-import java.util.List;
 
 // This class exists as template for code generation and for testing
 public class TwoChannelJoinProbe
```

Comment:
I don't get the purpose of this class. Isn't `SimpleJoinProbe` a sufficient template for code generation? @dain?

Reply:
IIRC the multi-channel probing is pretty different from the simple case, but if they are the same now, we could drop this.